packages/create/README.md

```diff
@@ -26,6 +26,11 @@ Calling `sap-ux add html` will add html files for local preview and testing to t
 ```sh
 sap-ux change add html ui5-test.yaml
 ```
+### annotations-to-odata
```

Comment:
Please rename it to just `annotations`. `sap-ux add annotations` is totally fine. OData is a give in the context of Fiori apps.

Reply:
Renamed with: 5a50c963a83e1bb04fb9fabce3b9dec051eb587a